src/Runner.Listener/Configuration/ConfigurationManager.cs

```diff
@@ -522,15 +521,21 @@ private async Task<bool> IsHostedServer(string serverUrl, VssCredentials credent
 
         private async Task<GitHubAuthResult> GetTenantCredential(string githubUrl, string githubToken)
         {
-            var gitHubUrl = new UriBuilder(githubUrl);
-            var githubApiUrl = $"https://api.{gitHubUrl.Host}/repos/{gitHubUrl.Path.Trim('/')}/actions-runners/registration";
+            var gitHubUrlBuilder = new UriBuilder(githubUrl);
+            var githubApiUrl = $"https://api.{gitHubUrlBuilder.Host}/actions/register-runner";
```

Comment:
is this new api ready for production usage?
if not, then you probably want to keep the `GetTenantCredential()` as it is and add a new method, like: `GetTenantCredentialPreview()`, and flip between these 2 bases on some env variable (GITHUB_NEW_REGISTERENDPOINT=true) to test the new endpoint.

Reply:
It'll be in production tomorrow. I just created this little early, so that you can have a look. Will merge it once the api is in production.